app/code/Magento/MediaGalleryUi/Test/Mftf/ActionGroup/AdminMediaGalleryFolderSelectByFullPathActionGroup.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminMediaGalleryFolderSelectByFullPathActionGroup"
```

Comment:
I think the extended action group is changed fundamentally. I would have independent `AdminMediaGalleryFolderSelectByFullPathActionGroup` not extending the `AdminMediaGalleryFolderSelectActionGroup`.

Reply:
created as a separate action group that not extends AdminMediaGalleryFolderSelectActionGroup